CHANGELOG.md

```diff
@@ -6,14 +6,19 @@
 
 #### New roles
 
-  - addons/Lmod: allow to install Lmod and specify custom modulefiles path
-
+  - addons/Lmod: allow to install Lmod and specify custom modulefiles path (#390)
+  - addons/singularity: allow to instal Singularity (#403)
+  
 #### Roles improvement
 
   - core/ssh_master:
-    - add ssh jump capability in multi icebergs context (one level only)
+    - add ssh jump capability in multi icebergs context (one level only) (#395)
   - addons/nic_nmcli:
-    - adapt to new inventory format
+    - adapt to new inventory format (#401)
+  - core/hosts_file:
+    - prevent many blanck files when hosts without network_interfaces (#406)
+  - core/rsyslog_server/client:
```

Comment:
I misread it at first.
 \- core/rsyslog_{server|client}: ?
 \- core/rsyslog_server and core/rsyslog_client: ?

I prefer the latter since a search for `rsyslog_client` in the changelog will find this change.